hal/targets.json

```diff
@@ -1823,7 +1823,7 @@
         "inherits": ["Target"],
         "core": "Cortex-M0+",
         "default_toolchain": "uARM",
-        "supported_toolchains": ["GCC_ARM", "uARM"],
+        "supported_toolchains": ["GCC_ARM", "uARM", "IAR"],
```

Comment:
I missed this the first time around, but the release version `"5"` can only be added to targets that support `GCC_ARM`, `IAR`, and `ARM`. Can the `ARM` toolchain be added here?